src/libraries/System.DirectoryServices.Protocols/src/CompatibilitySuppressions.xml

```diff
@@ -0,0 +1,67 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- https://learn.microsoft.com/dotnet/fundamentals/package-validation/diagnostic-ids -->
+<Suppressions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
```

Comment:
This is OK for now - but if we wanted to make this more common we could let projects opt out of this setting - https://github.com/dotnet/runtime/blob/b61c07c1436b5057f2d658f232211c0032202a95/eng/packaging.targets#L41